client/src/index.jsx

```diff
@@ -16,46 +16,40 @@ text-transform: none;
 letter-spacing: 0;
 `;
 
-//TO DO: fix sizing problem to make arrow look like an arrow
-const ArrowLeft = styled.div`
-background-position: left;
-left: 5px;
-width: 7px;
-height: 15px;
-top: 48px;
-background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAPCAYAAADUFP50AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAJlJREFUeNqc0k0KgCAQBlCdI0SLlkEtu4DLTt6yC7SN2hX0cwWbCQvDMa0PBhN9EsxIrbXgMs57RWueJR13Di+ooboeCEILpaZYDC/oCoshgLwYIhCL5TBtMcjOilUT1OJH6FdLrOWDobslYIN7/FCRmO4oMmCmIwbf6NGOAH4gZwA82EFnO7ghx14VuLRm6yAvtLDgEOUQYADt6VgCZRDsZgAAAABJRU5ErkJggg==) no-repeat top;
-position: absolute;
-padding: 0;
-margin: 0;
-display: block;
-`;
+let history = createBrowserHistory();
+let location = history.location;
+let action = history.action;
```

Comment:
I am not sure if history API actions will ever be used for our services, since when the proxy server URL changes, the whole page is refreshed and your history tracking is lost for this component instance (since a new component instance is requested by the proxy on refresh). History API and action tracking would work great, on the other hand, if anyone's proxy ever becomes a Single Page App instead, but that probably won't happen and isn't needed for BMR anyway.